app/src/main/java/com/example/rent/application1/models/Picture.java

```diff
@@ -0,0 +1,96 @@
+package com.example.rent.application1.models;
+
+import android.media.Image;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Picture implements Parcelable{
```

Comment:
Jeśli już masz dorzuconą bibliotekę `Lombok` to tu jest idealny przykład jej użycia. Pamiętaj, że jest taka adnotacja jak `@Data` https://projectlombok.org/features/Data